autorest/codegen/serializers/builder_serializer.py

```diff
@@ -615,6 +615,15 @@ def _create_body_parameter(
         body_kwarg_name = builder.request_builder.parameters.body_parameter.client_name
         if isinstance(body_param.type, BinaryType):
             retval.append(f"_{body_kwarg_name} = {body_param.client_name}")
+            if body_param.check_content_types:
+                content_types = "'" + "', '".join(body_param.check_content_types) + "'"
+                retval.extend(
+                    [
+                        "if not content_type:",
+                        f'    raise Exception("Please input content_type to indicate the kind of IO type. '
```

Comment:
If the check is required, should we use a concrete exception e.g. `ValueError`?

Reply:
same with this comment: https://github.com/Azure/autorest.python/pull/1269#discussion_r886216806